src/main/java/uk/gov/hmcts/darts/common/repository/TransformedMediaRepository.java

```diff
@@ -49,13 +50,14 @@ public interface TransformedMediaRepository extends JpaRepository<TransformedMed
     List<TransformedMediaDetailsDto> findTransformedMediaDetails(Integer userId, boolean expired);
 
 
+    @EntityGraph(attributePaths = {"mediaRequest.currentOwner.securityGroupEntities"})
```

Comment:
Not a comment for this PR, but I didn't realise this was a thing. Is this something we could be using more broadly to better control performance? Something worth exploring some best practices around?

@cmagowan-hmcts you aware of this one?

Reply:
Yeah it is something we should have probably employed more, I've not used it much before but, there maybe other use cases other than forcing lazy loading outside of transactions.  Worth looking into.